controllers/noderefutil/providerid.go

```diff
@@ -25,14 +27,17 @@ import (
 
 var (
 	// ErrEmptyProviderID means that the provider id is empty.
+	// Deprecated: This var has no function and is going to be removed in a future release.
 	ErrEmptyProviderID = errors.New("providerID is empty")
 
 	// ErrInvalidProviderID means that the provider id has an invalid form.
+	// Deprecated: This var has no function and is going to be removed in a future release.
```

Comment:
```suggestion
	// ErrInvalidProviderID means that the provider id has an invalid form.
	//
	// Deprecated: This var has no function and is going to be removed in a future release.
```

https://github.com/kubernetes-sigs/cluster-api/pull/7397/files
(kudos to @killianmuldoon )

Reply:
do all the deprecation comments need the leading `\\` (plus newline) or just the var comments?

Reply:
I think all of them, but I don't know

Reply:
Easy to do it for all of them, stand by

Reply:
done